dts/bindings/arm/arm,ethos-u.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Arm Limited and/or its affiliates <open-source-office@arm.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description:
+    The Arm Ethos-U is a micro NPU that enables neural networks to be hardware
+    accelerated on embedded devices. The Ethos-U NPU driver is provided as a
+    Zephyr module that is hooked up into the Tensorflow Lite Micro framework
+    as a 'TFLM custom operation'.
+    
+    This device tree entry defines board specific properties like the register
+    map address and interrupt line, and is therefor expected to be included
```

Comment:
Thank you, this description is excellent.

Typo: therefore

Reply:
Done.